props/prop_base.h

```diff
@@ -668,6 +672,27 @@ class PropBase : CommandParser, Looper, protected SaberBase, public ModeInterfac
 #endif
   }
 
+  void FindBladeAgain() {
```

Comment:
Looks like this was moved?
(Can we undo that to make the diff smaller?)

Reply:
Yes I just moved it to group it with the other relative functions. I can move it back and add that last piece of puzzle tomorrow. Thanks.

Reply:
moved back